centos/8/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:8
+MAINTAINER Alexey Kopytov <akopytov@gmail.com>
+
+# Fix missing locales
+ENV LANGUAGE="en_US.UTF-8"
+
+# Enable extra repositories
+RUN yum -y install \
+    wget \
+    yum-utils \
+    epel-release
+RUN curl -s https://packagecloud.io/install/repositories/packpack/backports/script.rpm.sh | bash
+
+# Install base toolset
+RUN yum -y groupinstall 'Development Tools'
+RUN yum -y install \
+    cmake \
+    sudo
+
+# Install python2 for testing
+RUN dnf -y install python2 python2-devel
+RUN pip2 install gevent
```

Comment:
Is it succeeds? Our backports repository does not have arm64 packages and I guess CentOS 8 itself does not have them too.

Reply:
Hm, actually it does succeed: https://hub.docker.com/layers/akopytov/packpack/centos-8-aarch64/images/sha256-ebb39865d0bb5a0a6261349125b2616e7f0a2d647e680baec8095978f945b0f0?context=repo

Reply:
Here gevent is installed from pip, not from the backports repository. I missed it.

Reply:
BTW, it was removed from the amd64 image: https://github.com/packpack/packpack-docker-images/pull/39#discussion_r338110004